frontend/src/pages/TermsOfUsePage/TermsOfUsePage.tsx

```diff
@@ -1,10 +1,10 @@
 import React, { Dispatch, SetStateAction } from "react";
 import Grid from "@material-ui/core/Grid";
-import styles from "./TermsOfServicePage.module.scss";
+import styles from "./TermsOfUsePage.module.scss";
 import { Typography } from "@material-ui/core";
 import { Helmet } from "react-helmet";
 
-function TermsOfServicePage(props: {
+function TermsOfUsePage(props: {
   setTabIndexFx?: Dispatch<SetStateAction<number>>;
```

Comment:
Will this be backwards compatible?

Reply:
no but I can add another route that will intercept any of the old routes (like `/termsofservice`) and reroute it to the new one `/termsofuse`. I think that is worth doing; I could also go as far as catching the params we used to use and rerouting to the new tabbed name but that seems less important... it'll just fall back in that case to the base page and the user clicking from outside would have to click in to "faq" or "contact" or whatever. But I think avoiding 404s is crucial

I'll look into catching the params; it probably won't be too hard and would be worth doing in case some superfan has bookmarked our team page or something

Reply:
Done

```
                    {/* redirect the old URL for possible outside links */}
                    <Route path={`/termsofservice`}>
                      <Redirect to={TERMS_OF_USE_PAGE_LINK} />
                    </Route>
```